set.mm

```diff
@@ -57021,6 +57071,15 @@ empty set when it is not meaningful (as shown by ~ ndmfv and ~ fvprc ).
       CIFABJKLCABMCABNO $.
   $}
 
+  ${
+    rnfvprc.y $e |- Y = ( F ` X ) $.
+    $( The range of a function value for a proper class is empty.
```

Comment:
"of a function value at"
or maybe "of the value of a function at"

Reply:
Comment changed accordingly.